src/MoviesContainer/MoviesContainer.js

```diff
@@ -1,21 +1,24 @@
 import React from 'react';
-import MovieCard from '../MovieCard/MovieCard'
+import MovieCard from '../MovieCard/MovieCard';
+// import SingleMovie from '../SingleMovie/SingleMovie';
 import './MoviesContainer.css';
 
-const MoviesContainer = ({movies}) => {
+const MoviesContainer = ({movies, movieClicked, displaySingleMovie}) => {
     const movieCards = movies.map(movie => {
         return (
             <MovieCard 
                 id={movie.id}
                 key={movie.id}
                 image={movie.poster_path}
+                backdrop={movie.backdrop_path}
                 title={movie.title}
                 rating={movie.average_rating}
+                displayMovieDetails={displaySingleMovie}
+                movieClicked={movieClicked}
```

Comment:
OH! We no longer need movieClicked. We can delete that on the next branch though.

Reply:
Cool, let's take that out.